spec/Section 2 -- Language.md

```diff
@@ -251,13 +251,13 @@ and validated to allow client tools to represent many GraphQL uses which may
 appear across many individual files.
 
 If a Document contains only one operation, that operation may be unnamed or
-represented in the shorthand form, which omits both the query keyword and
-operation name. Otherwise, if a GraphQL Document contains multiple
-operations, each operation must be named. When submitting a Document with
-multiple operations to a GraphQL service, the name of the desired operation to
-be executed must also be provided.
+represented in the shorthand form, which omits both the query keyword (for
```

Comment:
I find this backwards formulation confusing. First the shorthand is defined, and only afterwards it’s mentioned that it only works for queries. Why not say that first? Something like this maybe

“[...] If that single operation is a query, both the query keyword and the name may be omitted.”

But maybe this section could even be deleted, because it’s mentioned below in “Query shorthand” again. But more importantly, here it’s even inaccurate, because it doesn’t mention that the query must also not have variables or directives, otherwise the shorthand can’t be used.

Reply:
I think this should be proposed via a separate PR; let's keep this one focussed on just the task at hand :+1: